docs/src/managing-deis/configuring-dns.md

```diff
@@ -18,16 +18,67 @@ Apps can then be accessed by browsers at `appname.myapps.com`, and the controlle
 
 These records are necessary for all deployments of Deis other than Vagrant clusters.
 
-## Using xip.io
+## DNS without a Load Balancer
 
-For local Vagrant clusters or for dev clusters, you can use [xip][] to reference the IP of your load balancer. For example:
+On some platforms (Vagrant, for instance), a load balancer is not an easy or practical thing to
+provision. In these cases, one can directly identify the public IP of a Kubernetes node that is
+hosting a router pod and use that information to configure DNS or the local `/etc/hosts` file.
 
-    $ deis register http://deis.10.21.12.2.xip.io
+You can find the IP address of a node using `kubectl`:
 
-You would then create the cluster with `10.21.12.2.xip.io` as the cluster domain.
+```
+$ kubectl get pods --namespace=deis | grep deis-router
+deis-router-ih25q           1/1       Running   0          2h
+```
+
+Using the pod name determined through the above, one can
+use `kubectl` to determine what node the pod is running on:
+
+```
+$ kubectl describe pod deis-router-ih25q --namespace=deis
+```
+
+The `Node` field of the response should provide an IP:
+
+```
+Node:				10.245.1.3/
+```
+
+If the IP is public, this can be used to configure DNS or the
+local `/etc/hosts` file.
+
+An easy way to configure wildcard DNS is to use [xip.io][] to reference the IP of the node running
+your router. For example:
+
+    $ deis register http://deis.10.245.1.3.xip.io
 
 Note that xip does not seem to work for AWS ELBs - you will have to use an actual DNS record.
 
+Alternatively, in your `/etc/hosts` file, add an entry like this:
+
+```
+10.245.1.3    example.com deis.example.com
+```
+
+This will get you started, though you may find that you have to manually maintain this file as you
+create and deploy more applications to the cluster.
+
+## Testing
+
+To test that traffic reaches its intended destination, a request can be
+sent to the Deis controller like so:
+
+```
+curl http://deis.<platform domain>/v2/
```

Comment:
this was a tie-over from the old docs (see https://github.com/deis/workflow/pull/215/files#diff-3122c85f28ce9c670a6d843bc29638feL191). I'm more than happy to change it to example.com though :)